scripts/lint.sh

```diff
@@ -89,7 +89,7 @@ while getopts 'cwdlgh' opt; do
         SELF_ARGS_WITHOUT_DOCKER+="-l "
         ;;
     g)
-        GOLANGCI_LINT_ARGS+="--out-format=github-actions "
+        GOLANGCI_LINT_ARGS+=""
```

Comment:
Why are we doing this?

Reply:
New versions of golanci-lint raise a warning that this flag is deprecated. That said it does look weird to add an empty string here, so probably the correct change is to delete the whole line.

Reply:
Removed that line as suggested